### ISSUE TITLE
Download not working from Chrome

### ISSUE THREAD

**Issue by neuhaus** [kern/filepizza#55]
I'm on Chrome 54 on Linux (connected to the internet via NAT).
I upload a file and visit the URL on another browser on firefox 48 running on the same machine.
I see the filename and size and click "download" in firefox.

Nothing happens.

**Comment by k0nsl** [comment]
I am in the exact same situation. It shows the `filename` and `filesize` but it does not download. I have tried seeding in Google Chrome and Mozilla Firefox to no avail.

**Comment by kern** [comment]
I've updated the [WebTorrent](https://github.com/feross/webtorrent) version to the latest available and can successfully upload/download files in Chrome. Can you confirm @neuhaus @k0nsl ? :)

**Comment by YanMoou** [comment]
I have the same issue, and I have  the lasted version of WebTorrent but it still can`t download any file.
the Chrome print this error message in Console :`WebSocket connection to 'wss://tracker.webtorrent.io/' failed: Error in connection establishment: net::ERR_NAME_NOT_RESOLVED`
I don`t know if this would help you. 
Besides, I find that when I use file.pizza to transfer a file ,the  `https://file.pizza/socket.io/?EIO=3&transport=polling&t=LUoUZ1U&sid=xxxxxxxxxxxxx` will not stop send request until the file has been delivered. But I build the same service on my own machine, it will only send two times request and it stop, I want to know if something wrong with it.  Thanks.

**Comment by YanMoou** [comment]
I have the same issue, and I have  the lasted version of WebTorrent but it still can/`t download any file.
the Chrome print this error message in Console : `WebSocket connection to /'wss://tracker.webtorrent.io/' failed: Error in connection establishment: net::ERR_NAME_NOT_RESOLVED`
I don`t know if this would help you. 
Besides, I find that when I use file.pizza to transfer a file ,the brower will not stop send request  `https://file.pizza/socket.io/?EIO=3&transport=polling&t=LUoUZ1U&sid=xxxxxxxxxxxxx`  until the file has been delivered. But I build the same service on my own machine, it will only send two times request and it stop, I want to know if something wrong with it.  Thanks.

**Comment by neuhaus** [comment]
I tested the website, did not fork the project and install it locally.

Results:
On Linux, i upload a file from firefox and try to download it with chrome and Opera. The animation starts but the transfer stalls at 0 bytes and 0 peers.

Opera dumps out this error message:
> [1011/104844:ERROR:webrtcsession.cc(1226)] ConnectDataChannel called when data_channel_ is NULL.


Next, I try to upload the file with Chrome and download with Firefox and Opera.
Firefox: download hangs at 0 bytes 0 peers
Opera: download **works**!

On the console in firefox I messages about Firefox being unable to open a WSS:// connection to various trackers.

Perhaps my adblocker is preventing this from working? I'm using ublock origin.

**Comment by k0nsl** [comment]
@kern: I installed it on a VPS just some minutes ago and it appears `tracker.webtorrent.io` isn't functional at the moment. No DNS record found.
And FF says:

```
Firefox can’t establish a connection to the server at wss://tracker.webtorrent.io/.
```

**Comment by kern** [comment]
Fixed along with #56.
